### ISSUE TITLE
Not presenting all Desktop files in the panel

### ISSUE THREAD

**Issue by scallaway** [home-sweet-gnome/dash-to-panel#2091]
I'm observing Dash to Panel not reading the appropriate Desktop files for certain programs (mainly games or game stores) when they're present in the filesystem.

I'm able to see the icon for the game that I want to open in Arc Menu, as well as [Gnome Shell Extension App Indicator](https://github.com/ubuntu/gnome-shell-extension-appindicator) - when the programs are running.

I have a feeling that Dash to Panel doesn't have a wide enough reach on the directories it searches in when trying to find Desktop files - although I'm not 100% sure on this (and it could be an App ID issue as well maybe?).

**Linux distribution and version**
`Linux 6.8.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 17 Apr 2024 15:20:28 +0000 x86_64 GNU/Linux`

**GNOME Shell version**
46.1

**Dash-to-Panel version**
62

**Where was Dash-to-Panel installed from?**
The GNOME extensions website

**Screenshots / Video captures**
![image](https://github.com/home-sweet-gnome/dash-to-panel/assets/16577635/d686d469-233b-442c-b866-96d3b982e733)
> Guild Wars 2 game showing in Arc Menu with icon

![image](https://github.com/home-sweet-gnome/dash-to-panel/assets/16577635/2d1370aa-e276-4001-9773-b0f3bb2506f3)
> Guild Wars 2 running, but no icon in the panel

**Additional Info**
This obviously happens for more than just GW2, but that was the easiest game to show.

**Comment by charlesg99** [comment]
What does the icon look like on the default gnome dash in the overview? Dash to panel doesn't fetch icon files, we use what is provided by gnome.

**Comment by scallaway** [comment]
> What does the icon look like on the default gnome dash in the overview?

Yeah it looks the same as what I was showing in the original post - a rectangle with a cog.

Does this mean that GNOME isn't setup correctly to find those icons?

**Comment by scallaway** [comment]
For what it's worth, I can see the icons when I search for the game in either Arc Menu or the usual Applications Search (without extension enabled).

It's just when I open the program, it doesn't seem to find the icon for the Dash or for Dash to Panel
